.eslintignore

```diff
@@ -0,0 +1,4 @@
+**/flow-typed/**
```

Comment:
probably don't need the `**` prefix here since this isn't a lerna monorepo

Reply:
or you still like it?

Reply:
forgot to fix